prebid-mobile/adops-setup-rewarded-video-mopub.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: page_v2
+title: Setup Rewarded Video for MoPub
+description: Setup Rewarded Video for MoPub
+pid: 1
+top_nav_section: prebid-mobile
+nav_section: prebid-mobile-adops
+sidebarType: 3
+---
+
+# Step-by-Step Line Item Setup for Rewarded Video on MoPub
+
+* TOC
+{:toc }
+
+This page describes step-by-step instructions to set up rewarded-video line items on MoPub to be used with the Prebid Mobile SDK.  
+
+## Step 1. Add a line item  
+
+  In the *Add a line item* section:
+  1.  For the *Type and Priority* settings, select *Non-Guaranteed* as the type and set the priority to twelve. This ensures the line item will compete with all other demand.  
+  2. Set the Rate to the price you want to target.&nbsp;  
+  <img src="/assets/images/ad-ops/mopub/rewarded/mopub-lineitem-rewarded.png" width="1000" height="544">&nbsp;&nbsp;  
+  3. In the *Advanced Targeting* section, set the target for *Keywords* to `hb_pb:0.50`&nbsp;
```

Comment:
We're only setting it to hb_pb:0.50 because we set the Rate to .50. Maybe clarify with something like this: "...set the target for Keywords to include the specified Rate: hb_pb:0.50"